crates/sui-adapter-transactional-tests/tests/sui/freeze.move

```diff
@@ -3,12 +3,76 @@
 
 // test that freezing prevents transfers/mutations
 
-//# init --accounts A
+//# init --addresses test=0x0 --accounts A
```

Comment:
@tnowacki, could you help me figure out how to fix this test? I am copying what is done in the other publishing tests and think I did it right, but am getting

```
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: DuplicateObjectRefInput', crates/sui-transactional-test-runner/src/test_adapter.rs:472:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
NVM, I figured it out--just needed to update those object #'s.